cmd/minikube/cmd/root.go

```diff
@@ -84,8 +83,20 @@ var RootCmd = &cobra.Command{
 // Execute adds all child commands to the root command sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	defer audit.Log(time.Now())
-
+	audit.Mutex.Lock()
```

Comment:
We should do this locking inside `LogCommandStart` and `LogCommandEnd`, that way if we call it somewhere else we don't have to repeat this locking.